endtoend/evaluators/slashing.go

```diff
@@ -169,14 +170,105 @@ func insertDoubleAttestationIntoPool(conns ...*grpc.ClientConn) error {
 			Data:            attData,
 			Signature:       privKeys[committee[i]].Sign(signingRoot[:]).Marshal(),
 		}
-		for _, conn := range conns {
-			client := eth.NewBeaconNodeValidatorClient(conn)
-			_, err = client.ProposeAttestation(ctx, att)
-			if err != nil {
-				return err
-			}
+		client := eth.NewBeaconNodeValidatorClient(conns[0])
+		_, err = client.ProposeAttestation(ctx, att)
+		if err != nil {
+			return err
 		}
 		slashedIndices = append(slashedIndices, committee[i])
 	}
 	return nil
 }
+
+func proposeDoubleBlock(conns ...*grpc.ClientConn) error {
```

Comment:
Maybe we can return more informative errors in each of the conditionals in this func, something like errors.Wrap(err, "could not do X")